lib/zendesk_apps_support/app_version.rb

```diff
@@ -31,7 +38,11 @@ def valid_for_update?
     end
 
     def deprecated?
-      @version == DEPRECATED
+      DEPRECATED.include?(@version)
```

Comment:
Allows for `DEPRECATED` to be an array of values. Or, when 1.0 is deprecated, will we just drop 0.5 from the list entirely? If the latter is true, then this can be removed.